lm_reinvest.py

```diff
@@ -0,0 +1,122 @@
+from audioop import add
+import json
+import logging
+import sys
+from time import sleep
+
+import utils
+from utils import rpc, waitForTx
+
+'''
+Script to keep reinvesting in DFI-DUSD pool every time minReinvest is reached on your address.
+Based on kuegi's DefiVaultMaxi.py script
+
+kuegi DFI donations: dLBqjysPVXYQX4dFSp5hMWdVfbdeY4aHVS
+krysh DFI donations: dZ69fTXJ15YyDKCjAxTKqJ9qx2iV5Yq7cS
+'''
+
+errortimeout = 5  # blocks
+
+address = "myAddress"
+min_reinvest = 3 # measured in DFI
+
+lm_pair = "DUSD-DFI"
+log_to_console = True
+log_to_file = False
+log_id = ""
+
+settings_path = sys.argv[1] if len(sys.argv) > 1 else None
+# read settings
+if settings_path is not None:
+    print("Importing settings from %s" % settings_path)
+    with open(settings_path) as f:
+        settings = json.load(f)
+        utils.NODE_USER = settings['NODE_USER']
+        utils.NODE_PASSWORD = settings['NODE_PASSWORD']
+        address = settings['address']
+        min_reinvest = settings['minReinvest']
+
+        if "logToFile" in settings:
+            log_to_file = settings['logToFile']
+        if "logToConsole" in settings:
+            log_to_console = settings['logToConsole']
+        if "logId" in settings:
+            log_id = settings['logId']
+            utils.logId = log_id
+        if "telegram" in settings:
+            utils.TELEGRAM_TOKEN = settings['telegram']['token']
+            utils.TELEGRAM_CHANNEL = settings['telegram']['channel']
+
+logger = utils.setup_logger("lm-reinvest" + log_id, logging.INFO, log_to_console, log_to_file)
+utils.LOGGER = logger
+
+assets = lm_pair.split("-")
+asset_a = assets[0]
+asset_b = assets[1]
+
+utils.send_telegram(f"starting to reinvest in DFI-DUSD")
+logger.info(f"starting to reinvest in pair {asset_a}-{asset_b}. Will reinvest if DFI reaches {min_reinvest}")
```

Comment:
telling telegram to do DFI-DUSD but actually using pair from variable? not cool reporting ;)
I mean you check later that it is DUSD-DFI (cause thats the only pair with assetA == DUSD) but still weird to read like that